cedar-policy/tests/public_interface.rs

```diff
@@ -198,15 +198,15 @@ fn authorize_custom_request() -> Result<(), Box<dyn Error>> {
         Response::new(Decision::Allow, [alice_view_id].into(), Vec::new())
     );
 
-    // request with Account::"jane" and an unspecified action
+    // request with Account::"jane" and the default entity
     let principal = EntityUid::from_type_name_and_id(
         EntityTypeName::from_str("Account").unwrap(),
         EntityId::from_str("jane").unwrap(),
     );
     let request3 = Request::new(
-        Some(principal.clone()),
-        None,
-        Some(resource.clone()),
+        principal.clone(),
+        r#"__cedar::"Default""#.parse().unwrap(),
```

Comment:
Is this going to work with RFC52?

Reply:
Why wouldn't it?

Reply:
It uses `__cedar`?

Reply:
Did we decide the interactions of those RFCs?  I.e., whether `__cedar::Default` will be one of the allowed uses of the `__cedar` namespace (like `__cedar::Long`)?

Reply:
It has to be or we need to change the RFC. I was assuming it was because otherwise that part of the RFC doesn't make sense in the context of 52

Reply:
The `__cedar` syntax if only mentioned in the Alternatives section of RFC 55, not in the main proposal. But the intention in the alternatives was that `__cedar::Default` would be builtin the same way as `__cedar::Long`.

Reply:
(Regardless, I think it should be removed from the tests here since we decided not to support it as part of RFC 55.)